pkg/monitor/intervalcreation/rendering.go

```diff
@@ -125,6 +125,14 @@ func BelongsInKubeAPIServer(eventInterval monitorapi.EventInterval) bool {
 	return true
 }
 
+func isNodeLifecycle(eventInterval monitorapi.EventInterval) bool {
```

Comment:
Be sure this doesn't exist elsewhere.  Perhaps as a `NodeFrom` function that you could len check inline where you use this.